components/dashboard/src/components/Delayed.tsx

```diff
@@ -0,0 +1,26 @@
+/**
+ * Copyright (c) 2023 Gitpod GmbH. All rights reserved.
+ * Licensed under the GNU Affero General Public License (AGPL).
+ * See License.AGPL.txt in the project root for license information.
+ */
+
+import { FC, useEffect, useState } from "react";
+
+type Props = {
+    wait?: number;
+};
+export const Delayed: FC<Props> = ({ wait = 1000, children }) => {
```

Comment:
**suggestion**: Not sure if this is the best line to comment about this or if this is already implementing the following: Ideally, we would wait a second or two before showing the full-page loading indicator to avoid having spinners or loading states everywhere. 💡

Reply:
Yup, I think the same thing. This component is built so we can use it for those loading indicators too, and just wrap them with this. We can do it in the components themselves so we get it for free everywhere..